src/v2/Components/UserSettings/TwoFactorAuthentication/Components/BackupSecondFactorReminder.tsx

```diff
@@ -0,0 +1,36 @@
+import { BorderBoxProps, Box, Flex, Sans } from "@artsy/palette"
+import React from "react"
+
+interface BackupSecondFactorReminderProps extends BorderBoxProps {
+  backupSecondFactors: string[]
+  factorTypeName: string
+}
+
+export const BackupSecondFactorReminder: React.FC<BackupSecondFactorReminderProps> = props => {
+  const { backupSecondFactors, factorTypeName } = props
+
+  return (
+    <Box minHeight="280px">
+      <Sans size="3" color="black60">
```

Comment:
The `Sans` component is deprecated and shouldn't be used. You can [use `Text` and V2 variants here](https://palette-storybook.artsy.net/?path=/story/components-text--variants)

Reply:
Noted. I've changed them in this Components and have added a chore task on the Epic to switch on the other 2FA Components to use `Text`.